### PR TITLE
e2e: increase the size of header to avoid flaky

### PR DESCRIPTION
**Pull request by zirain**
Fixes: https://github.com/envoyproxy/gateway/issues/2720

### CODE REVIEW

**Comment by zirain** [comment]
failed to set a small timeout, see #2744 for more info.

File: test/e2e/tests/client_timeout.go [hunk 1]
```diff
@@ -24,7 +24,8 @@ func init() {
...
...eHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValueFakeHeaderValue" +
```
Comment (arkodg) [inline test/e2e/tests/client_timeout.go:27]: can you generate this in a for loop instead :) ?

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Pull request merged**